hw_13042018/proc.rb

```diff
@@ -0,0 +1,12 @@
+def square_of_sum (my_array, proc_square, proc_sum)
+  sum = proc_sum.call(my_array)
+  proc_square.call(sum)
+end
+
+proc_square = Proc.new{|s| s *= s}
+proc_sum = Proc.new {|my_array|
+  my_array.inject{|s, num| s += num}
```

Comment:
Need a space after `{` and before `}`
=> `my_array.inject { |s, num| s += num }`